src/test/test-cryptolib.c

```diff
@@ -7,27 +7,19 @@
 TEST(string_hashsum) {
         _cleanup_free_ char *out1 = NULL, *out2 = NULL, *out3 = NULL, *out4 = NULL;
 
-        assert_se(string_hashsum("asdf", 4,
-                                 "SHA224",
-                                 &out1) == 0);
+        ASSERT_EQ(string_hashsum("asdf", 4, "SHA224", &out1), 0);
```

Comment:
This returns on negative errno, which is most common pattern in our code base. For such cases, please use `ASSERT_OK()`.